app/assets/javascripts/backbone/views/editable_table_view.js.coffee

```diff
@@ -2,22 +2,46 @@ class MS.Views.EditableTableView extends Backbone.View
 
   # This view may control multiple elements at once.
   # The view is called from the loan questionnaires view.
-  el: "table.editable-table"
 
-  editableTableInit: (e) ->
-    $('.editable-table tbody').sortable({
+  events:
+    'click tr [data-action="delete"]': 'removeRow'
+    'click .actions [data-action="add"]': 'addRow'
+    'change .editable-table[data-table="fixed_costs"] input.amount': 'totalFixedCostsChanged'
+
+  initialize: (e) ->
+    @$el.find('tbody').sortable({
       handle: "[data-action='move']"
     })
+    @parent = e.parent
+
+    console.log({totalFixedCosts: @totalFixedCosts()})
+
+    products = @$("tr[data-group='product']:not('.hidden')").map (index, productRow) =>
+      new MS.Views.BreakevenProductView(el: productRow)
+
+    @$("tr[data-group='product-total']").map (index, totalRow) =>
+      new MS.Views.BreakevenProductTotalView(el: totalRow, products: products)
+
+  totalFixedCosts: =>
```

Comment:
@linesarefuzzy @hooverlunch Any suggestions on how to get the `totalFixedCosts`? This method isn't going to work because it adds all fixed costs for all break even reports. I can't think of any good ways to do that without subclassing `editable_table` for each of the different tables, and then wiring the two up with events.